src/cr/cube/matrix/measure.py

```diff
@@ -945,23 +996,25 @@ def blocks(self):
     @lazyproperty
     def t_stats(self):
         """2D ndarray of float64 representing t-stats for means pairwise testing.
-
```

Comment:
This blank line should stay to separate the summary line from the continuation paragraph.